applications/RomApplication/python_scripts/calculate_rom_basis_output_process.py

```diff
@@ -170,6 +174,10 @@ def _PrintRomBasis(self, snapshots_matrix):
         rom_basis_dict["rom_settings"]["petrov_galerkin_number_of_rom_dofs"] = 0
         #NOTE "petrov_galerkin_number_of_rom_dofs" is not used unless a Petrov-Galerkin simulation is called, in which case it shall be modified either manually or from the RomManager
 
+        # Create the folder if it doesn't already exist
+        if not os.path.exists(self.rom_basis_output_folder):
+            os.makedirs(self.rom_basis_output_folder)
```

Comment:
I strongly recommend to use `pathlib.Path` over `os.path` (basically in modern python `os.path` should never be used)

Reply:
Done. Thanks.